src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj

```diff
@@ -53,10 +53,14 @@
     </MSBuild>
 
     <ItemGroup>
-      <PackageReference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.3.1.x86"
+      <!--
+        When updating AspNetCoreMajorVersion or AspNetCoreMinorVersion, add the following SiteExtension packages to
+        the list above and bump the version here.
+      -->
+      <PackageReference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.6.0.x86"
```

Comment:
This and the `@(PackageReference)` items in release/5.0 need to use `5.0`. Is that on your team @BrennanConroy❔

Reply:
Did you mean to put this comment on the other side of the diff?

I'll take a look at 5.0 and update 3.1 -> 5.0 if needed

Reply:
Much appreciated❕ Hopefully 3.1 isn't busted in this respect too.

Reply:
I don't think it is, that's the one that was fixed a couple months ago and I think the merge forward kept this as 3.1

Reply:
Technically 5.0 should probably be updated, but it's currently producing the correct package. I guess I'll update it for consistency.